services/speech_to_text/v1.js

```diff
@@ -85,18 +91,30 @@ module.exports = function (RED) {
 
       var s2t = function (audio, format, cb) {
         if (format !== 'wav' && format !== 'flac' && format !== 'ogg') {
-          var message = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
-          node.error(message, msg)
+          var message_err_format = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
+          node.error(message_err_format, msg)
           return;
         }
 
         if (format === 'ogg') format += ';codecs=opus';
 
-        var params = {
-          audio: audio,
-          model: model,
-          content_type: 'audio/' + format 
-        };
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2398731577/issues/source?bid=3225115&fileBranchId=3231190#l101)